CONTRIBUTING.md

```diff
@@ -10,11 +10,17 @@ The development environment is mainly up to the developer. Our recommendations a
 virtual environment and install the necessary requirements. Example
 
 ```sh
-python -m venv venv
-source venv/bin/activate
-python -m pip install -U pip
-python -m pip install -r requirements.txt
-python -m pip install -r dev-requirements.txt
+# Install and upgrade pip & poetry
+python -m pip install --upgrade pip poetry
+
+# Create virtualenv
+python -m poetry env use <PATH_TO_PYTHON_VERSION>
+
+# install package dependencies including dev dependencies
+python -m poetry install --dev
```

Comment:
This does not seem to be right, there is no `--dev` option (https://python-poetry.org/docs/cli/#install)

Reply:
Nice catch. Thank you